app/apps/itinerary/views.py

```diff
@@ -35,6 +35,11 @@ class ItineraryViewSet(ViewSet, GenericAPIView, DestroyModelMixin, CreateModelMi
     filter_backends = [DjangoFilterBackend]
     filterset_fields = ["created_at"]
 
+    def get_serializer_class(self):
+        print("get_serializer_class")
+        print(self.request.version)
```

Comment:
Ik vraag me af of we voor deze view een andere serialiser nodig hebben. In ieder geval goed om te weten dat we zo ook binnen de views de versies kunnen onderscheiden.

Reply:
Ik had deze als checker gebruikt. Inderdaad denk ik ook dat een andere serializer niet nodig is. We breiden de huidige serializer wel uit